### ISSUE TITLE
querydsl-bom missing jakarta classifier

### ISSUE THREAD

**Issue by jherkel** [querydsl/querydsl#3424]
## Observed vs. expected behavior

When tried to build project with spring boot 3.0.0 RC2 I found out that spring data jpa included jar files from querydsl-bom. But these jars 
don't contain classifier <jakarta>. Probably there must be two different versions of bom, one for "old" javax and a new one for "jakarta"

## Environment
java 17 & spring boot 3.0.0 RC2

Querydsl version:
5.0.0
JDK:
openjdk 17

**Comment by Deathef** [comment]
You can look at this plan.
#3421

**Comment by jherkel** [comment]
This is something else. I don't have problem with compilation or running. But I need to "rewrite" querydsl dependencies because they have wrong classifiers from querydsl-bom. So I think that we need something like queryds-jakarta-bom.

**Comment by wilkinsona** [comment]
I don't think two separate boms are required. The existing `querydsl-bom` should be able to manage the "standard" artifacts and those with the "jakarta" classifier side-by-side.

**Comment by jwgmeligmeyling** [comment]
Yes. a `querydsl-jakarta-bom` could be a solution in the mean time (although I also do consider dropping support for `javax.` apis in the next release anyway)

**Comment by abdulk1** [comment]
Hi. Any update here? Or an approximate timeline?

**Comment by Kashif-Emoldino** [comment]
Any update on this issue ?

**Comment by rcsilva83** [comment]
This is affects me too. Can I send you a pull request on this?
